test/others/test_utils.py

```diff
@@ -1245,6 +1246,21 @@ def greet2(name: str):
     assert greet2("John") == "Hello John"
 
 
+def test_secure_model_loading():
+    # setting the flag explicitly to zero
+    os.environ["TORCH_FORCE_WEIGHTS_ONLY_LOAD"] = "0"
```

Comment:
Instead of setting the environment variable, we should mock it. Otherwise it might affect other tests, etc. You can have a look at the [telemetry tests](https://github.com/deepset-ai/haystack/blob/e954230ae700327861133057455c2728d0dd9cc9/test/others/test_telemetry.py#L112) to see how you can do that. Basically, it comes down to sth like:
`monkeypatch.setattr(test_utils, "TORCH_FORCE_WEIGHTS_ONLY_LOAD", "1")`

Reply:
Super suggestion. Changed it :)